zip-1014.rst

```diff
@@ -209,13 +209,16 @@ Zcash ecosystem, to perform major ongoing development (or other work) for the
 public good of the Zcash ecosystem, to the extent that such teams are available
 and effective.
 
-The funds SHALL be received and administered by ZF. ZF MUST disburse them as
-"Major Grants", but subject to the following additional constraints:
+The funds SHALL be received and administered by ZF. ZF MUST disburse them for
+"Major Grants" and expenses reasonably related to the administration of Major 
+Grants, but subject to the following additional constraints:
```

Comment:
As originally drafted, ZIP 1014 restricts ZF from disbursing MG Slice funds for anything other than Major Grants. To be able to disburse funds from the Discretionary Budget on behalf of MGRC (aka ZCG), we need to relax this restriction.